.travis.yml

```diff
@@ -15,12 +13,15 @@ matrix:
         # Lint the first commit in the PR.
         - \[ -z "$TRAVIS_COMMIT_RANGE" \] || (echo -e '\nLinting the commit message according to the guidelines at https://goo.gl/p2fr5Q\n' && git log $TRAVIS_COMMIT_RANGE --pretty=format:'%h' --no-merges | tail -1 | xargs npx -q core-validate-commit --no-validate-metadata)
     - name: "Test Suite"
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-8
```

Comment:
Should we be building/testing with the minimum supported version ([4.9.4](https://github.com/nodejs/node/blob/master/BUILDING.md#supported-toolchains))?

Reply:
I mostly wanted to see if it's faster...
Since it's just for quick feedback, I thought if it was faster, we could give it a try. But it seems like 4.9 is faster.

Reply:
In terms of build speed, are you taking into account whether `ccache` is speeding up the build or whether the build needs to build everything from scratch when you switch compiler levels?

Reply:
I run it at least 3 times.
First time takes ~30+ minutes.
second and third take 15m, with GCC8 and clang. But only 13-14m with gcc4.9